deepmd/pt/utils/env_mat_stat.py

```diff
@@ -146,6 +146,9 @@ def iter(
                 radial_only,
                 protection=self.descriptor.env_protection,
             )
+            # apply excluded_types
+            exclude_mask = self.descriptor.emask(nlist, extended_atype)
+            env_mat *= exclude_mask.unsqueeze(-1)
```

Comment:
may be a better option to use the same mechanism as the atomic model 
https://github.com/deepmodeling/deepmd-kit/blob/cc30f275f34fd999658025446703289c4718aa31/deepmd/pt/model/atomic_model/dp_atomic_model.py#L209-L222

Reply:
After I check the code, I think the zeros generated by the descriptor exclude types will still be passed to the neural network, so I think my implementation is correct.
The zeros generated by `pair_exclude_types` seems not used for stat. Need to confirm whether it is correct.

Reply:
As for descriptors, the pair exclusion is not used in stat.